lrauv_ignition_plugins/test/helper/LrauvTestFixture.hh

```diff
@@ -213,6 +213,14 @@ class LrauvTestFixture : public ::testing::Test
         // FIXME: LRAUV app hangs after quit, so force close it
         // See https://github.com/osrf/lrauv/issues/83
         std::string bufferStr{buffer};
+
+        std::string error{"ERROR"};
```

Comment:
Thanks, @arjo129! I think we really only care about CRITICAL failures, since those will abort the mission

Reply:
Here we go: https://github.com/osrf/lrauv/pull/120